source/Calamari.Tests/KubernetesFixtures/ManifestReporterTests.cs

```diff
@@ -40,16 +41,21 @@ public void GivenValidYaml_ShouldPostSingleServiceMessage(string enabledFeatureT
             var memoryLog = new InMemoryLog();
             var variables = new CalamariVariables();
             variables.Set(KnownVariables.EnabledFeatureToggles, enabledFeatureToggle);
-
-            var yaml = @"foo: bar";
-            var expectedJson = "{\"foo\": \"bar\"}";
+            
+            //Test that quotes are preserved, especially for numbers
```

Comment:
Should we have ints and floats both in and out of quotes here for extra security?

Reply:
Good idea